### PR TITLE
Bug 1889956: CVE-2020-8564

### PR DESCRIPTION
**Pull request by coreydaley**
I was unable to cherry-pick #193 and had to create a new pull request manually.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,34 +4,32 @@ go 1.13
...
+	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
+	k8s.io/api v0.19.3
+	k8s.io/apimachinery v0.19.3
+	k8s.io/client-go v0.19.3
+	k8s.io/component-base v0.19.3
 	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.18.0
-	k8s.io/kubernetes v1.18.0
+	k8s.io/kubectl v0.19.3
+	k8s.io/kubernetes v1.19.3
```
Comment (adambkaplan) [inline go.mod:32]: Kube should not be bumped a y version. The correct fix version is 1.18.10 (`0.18.10` for most k8s.io bumps).

See https://github.com/kubernetes/kubernetes/issues/95622 for the list of fix versions upstream.

**Review by adambkaplan — CHANGES_REQUESTED** [review]
Kube bumps break a lot of things - we shouldn't rebase in a backport.

File: go.mod (hunk 1 above)
Comment (coreydaley) [reply go.mod]: Corrected in current push

**Comment by coreydaley** [comment]
/retest
VPC limit exceeded

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Comment by adambkaplan** [comment]
@bparees somehow Corey was able to green-button merge this. IMO this is a safe thing to keep, QE and patch managers just need to be aware.

**Pull request merged**